pySmartDL/download.py

```diff
@@ -4,16 +4,13 @@
 
 from . import utils
 
-def download(url, dest, startByte=0, endByte=None, headers=None, timeout=4, shared_var=None, thread_shared_cmds=None, logger=None, retries=3):
+def download(url, dest, requestArgs, startByte=0, endByte=None, timeout=4, shared_var=None, thread_shared_cmds=None, logger=None, retries=3):
```

Comment:
That's true that func download() is an internal function, but still i would like it to be generic and support not passing requestArgs at all (defaulting to None)